qutebrowser/browser/commands.py

```diff
@@ -277,16 +277,9 @@ def tab_pin(self, count=None):
         """
         tab = self._cntwidget(count)
         if tab is None:
-            return
+            raise cmdexc.CommandError("Tab {} does not exist!".format(count))
```

Comment:
All other commands just silently do nothing if the tab count is wrong. It's debatable whether that's a good idea, but only showing an error for `:tab-pin` seens inconsistent :wink: